stardog/admin.py

```diff
@@ -46,37 +45,45 @@ def shutdown(self):
         """
         self.client.post('/admin/shutdown')
 
-    #TODO
-    # def alive(self):
-    #     """
-    #     Determine whether the server is running
-    #     https://stardog-union.github.io/http-docs/#operation/aliveCheck
-    #     :return:
-    #     """
+    def alive(self):
+        """
+        Determine whether the server is running
+        https://stardog-union.github.io/http-docs/#operation/aliveCheck
+        """
+        r = self.client.get('/admin/alive')
+        alive = True if r.status_code == 200 else False
+        return alive
```

Comment:
cut out the middleman
```suggestion
        return r.status_code == 200
```